.gitignore

```diff
@@ -7,3 +7,4 @@ org.scala-ide.sdt.update-site/site.xml
 .idea
 .metadata
 *~
+org.scala-ide.sbt.build
```

Comment:
The change in this commit is not needed. The `sbt.build` dir is no longer used. You can remove it from your repo.